client/electron/src/setup.ts

```diff
@@ -213,6 +214,7 @@ export class ElectronCapacitorApp {
   }
 
   updateMountpoint(path: string): void {
```

Comment:
```suggestion
  createAndRegisterBaseMountpointFolder(path: string): void {
```

I suggest to rename the method to something that reflect what the method do and give more hint about the path value

Reply:
This method just informs the ElectronCapacitorApp that a new mountpoint path has been set. What the class does with that information is up to the class.
If I had named it `addMountpointToQuickAccess` before, now I would have to rename it `createFolderAndAddMountpointToQuickAccess`, and later on when we'd want to add a shortcut on the desktop for the mountpoint I would have to rename it `createFolderAndAddMountpointToQuickAccessAndAddDesktopShortcut`.